packages/fxa-auth-server/lib/senders/emails/templates/passwordResetAccountRecovery/en-US.ftl

```diff
@@ -0,0 +1,9 @@
+passwordResetAccountRecovery-subject = Password updated using recovery key
+passwordResetAccountRecovery-title = Your account password was reset with a recovery key
+passwordResetAccountRecovery-description = You have successfully reset your password using a recovery key from the following device:
+passwordResetAccountRecovery-action = Create new recovery key
+passwordResetAccountRecovery-regen-required = You will need to generate a new recovery key.
+passwordResetAccountRecovery-create-key = Create new recovery key:
+passwordResetAccountRecovery-email-change = This is an automated email; if you did not authorize this action, then <a data-l10n-name="passwordChangeLink">please change your password</a>.
```

Comment:
Do we need this here or can we pick this up from `automated-email-reset` fluent ID? Same question for `postAddAccountRecovery` FTL file.

You know this since you called it out on my PR but just to note here, you can reference that new `changePassword` txt partial merged in rather than setting `passwordResetAccountRecovery-email-change-plaintext`.